sklearn/model_selection/tests/test_search.py

```diff
@@ -2219,8 +2219,6 @@ def _more_tags(self):
     assert pairwise == cv._get_tags()["pairwise"], attr_message
 
 
-# TODO: Remove in 1.1
-@ignore_warnings(category=FutureWarning)
 def test_search_cv__pairwise_property_delegated_to_base_estimator():
     """
     Test implementation of BaseSearchCV has the _pairwise property
```

Comment:
The docstring should be update to mention the pairwise tag instead of the _pairwise property